src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String VALID_FILE_NAME = "src/test/resources/input.csv";
+
+    @Test
+    public void fileReaderServiceImpl_readFromValidFile_Ok() {
+        assertEquals(validReadFromFile(VALID_FILE_NAME),
+                new FileReaderServiceImpl().readFromFile(VALID_FILE_NAME));
```

Comment:
new FileReaderServiceImpl() - move it to field and initialize this field in BeforeClass method
same in all test classes